cmd/zc_filter.go

```diff
@@ -357,6 +357,54 @@ func (_ IncludeBeforeDateFilter) FormatAsUTC(t time.Time) string {
 	return formatAsUTC(t)
 }
 
+type softDeleteFilter struct {
+	deleteSnapshots bool
+	deleteVersions  bool
+}
+
+func (s *softDeleteFilter) DoesSupportThisOS() (msg string, supported bool) {
+	msg = ""
```

Comment:
You can do `return "", true`.

A bit of non-PR commentary; perhaps we should migrate DoesSupportThisOS into it's own interface, since it doesn't apply to all filters.